docs/resources/Channel.md

```diff
@@ -562,15 +562,17 @@ An object that represents a tag that is able to be applied to a thread in a `GUI
 
 ###### Forum Tag Structure
 
-| Field      | Type       | Description                                                                                                    |
-| ---------- | ---------- | -------------------------------------------------------------------------------------------------------------- |
-| id         | snowflake  | the id of the tag                                                                                              |
-| name       | string     | the name of the tag (0-20 characters)                                                                          |
-| moderated  | boolean    | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
-| emoji_id   | ?snowflake | the id of a guild's custom emoji \*                                                                            |
-| emoji_name | ?string    | the unicode character of the emoji \*                                                                          |
-
-\* At most one of `emoji_id` and `emoji_name` may be set.
+| Field      | Type      | Description                                                                                                    |
+| ---------- | --------- | -------------------------------------------------------------------------------------------------------------- |
+| id         | snowflake | the id of the tag                                                                                              |
+| name \*    | string    | the name of the tag (0-20 characters)                                                                          |
+| moderated  | boolean   | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
+| emoji_id   | snowflake | the id of a guild's custom emoji \*\*                                                                          |
+| emoji_name | ?string   | the unicode character of the emoji \*\*                                                                        |
+
+\* If you are modifying a `GUILD_FORUM` channel with tags in `available_tags`, for the new ones added, only this field is required.
+
+\*\* At most one of `emoji_id` and `emoji_name` may be set.
```

Comment:
Because a new PR would conflict with this one, can you make `emoji_id` and `emoji_name` both nullable and clarify the note?
```suggestion
| Field      | Type      | Description                                                                                                    |
| ---------- | --------- | -------------------------------------------------------------------------------------------------------------- |
| id         | snowflake | the id of the tag                                                                                              |
| name \*    | string    | the name of the tag (0-20 characters)                                                                          |
| moderated  | boolean   | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
| emoji_id   | ?snowflake | the id of a guild's custom emoji \*\*                                                                          |
| emoji_name | ?string   | the unicode character of the emoji \*\*                                                                        |

\* If you are modifying a `GUILD_FORUM` channel with tags in `available_tags`, for the new ones added, only this field is required.

\*\* At most one of `emoji_id` and `emoji_name` may be set to a non-null value.
```

Reply:
Didn't this got added in #5453? Without the note clarification

Reply:
It appears so, sorry. Thank you for keeping an eye out, I experienced this in the production build and found this PR while looking for duplicates for opening one myself. Going to mark this resolved